cmake/pip_utils.cmake

```diff
@@ -74,32 +74,38 @@ endfunction()
 
 function(pip_add_stub_target)
     set(options)
-    set(oneValueArgs NAME PACKAGE_NAME)
-    set(multiValueArgs DEPENDS)
+    set(oneValueArgs NAME PACKAGE_NAME ENABLE_SO_STUB)
+    set(multiValueArgs DEPENDS STUB_DEPENDS)
```

Comment:
DEPENDS will be passed to other targets if ENABLE_SO_STUB is false, while STUB_DEPENDS will not.
Same thing is done for HTML generation